kafka-watcher/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# librdkafka is not available in a stable alpine release
+from alpine:edge as kafka-watcher-builder
+
+# To force a rebuild, pass --build-arg REBUILD="$(DATE)" when running
+# `docker build`
+ARG REBUILD=1
+
+ARG WATCHER_REPO=https://github.com/monasca/monasca-watchers
+ARG WATCHER_BRANCH=master
+
+ENV GOPATH=/go
+ENV PROJECT_DIR=$GOPATH/src/github.com/monasca/monasca-watchers
+
+RUN apk add --no-cache openssl py2-jinja2
+
+RUN apk add --no-cache --virtual build-dep librdkafka-dev \
+      git go glide make g++ openssl-dev
```

Comment:
Why installing `make` and `openssl` in line 14 if you install `openssl-dev` here?

Reply:
Removed. Thanks